utils/sqlite_extract_data.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
```

Comment:
`use strict` is only needed on Node.js 4, which we're now dropping support for with the addition of better-sqlite3, right?